File: Pi_Day/Canale_Andrea/Canale_piday.c
#include<stdio.h>
float pigreco(){
int count=1;
    float pi=0;
    float divisore=1;
    for(int i=0;i<1000;i++){
        if(count%2==0){
            pi=pi-(4/divisore);
        }else{
            pi=pi+(4/divisore);
        }
        count++;
        divisore=divisore+2;
    }
    return pi;
}
int main(){
    printf("Il pigreco è %f", pigreco());
}

File: Pi_Day/Canale_Andrea/Canale_Piday.ts
function pigreco() {
    var count:number = 1
    var pi:number=0
    var divisore:number=1
    
    for (let index = 0; index < 1000; index++) {
        
        if(count%2==0){
            pi=pi-(4/divisore)
        }else{
            pi=pi+(4/divisore)
        }
        count++
        divisore=divisore+2
    }
    return pi 
}

console.log("Il pigreco è " + pigreco().toString())
